docs/theory-of-operation.md

```diff
@@ -0,0 +1,159 @@
+# apu features - theory of operation
+
+
+## Boot menu 
+
+It let selecting boot device (first options) and enter another menus to extended coreboot/seaBIOS features.  
+To enter boot menu Press `F10` key after reboot.   
```

Comment:
@piotr-kleins always wrap at 80 characters (I mean almost, obviously we do not wrap URLs since it breaks Markdown)